src/StructuralAnalyses/NonLinearStaticAnalyses.jl

```diff
@@ -49,7 +49,7 @@ function Base.show(io::IO, sa::NonLinearStaticAnalysis)
     show(io, sa.state)
 end
 
-"Solves an `NonLinearStaticAnalysis` `sa` with an AbstractSolver `alg`."
+"Solves an `NonLinearStaticAnalysis` problem with a given solver."
```

Comment:
I have a question, in general we want to use `  only for types or variable names too ?

Reply:
yes

Reply:
But i mean in the function docstrings, like this function apply the `BoundaryCondition` `bc` to the `Mesh` `m`. 
```julia
apply(m::Mesh, bc::BoundaryCondtion)
```